### PR TITLE
Upgrade to version 1.5.1

### PR DESCRIPTION
**Pull request by Mikaela**
There is also 1.5.2, but it appears macOS specific and I didn't see a .deb for it.

### CODE REVIEW

**Comment by SISheogorath** [comment]
Thanks a lot!

File: im.riot.Riot.json [hunk 1]
```diff
@@ -62,8 +62,8 @@
                 },
                 {
                     "type": "archive",
-                    "url": "https://github.com/vector-im/riot-web/releases/download/v1.5.0/riot-v1.5.0.tar.gz",
-                    "sha256": "2445ad7e6750a90394f783a3ffa41408bb73741247370bf1e7c33b66c7212923"
+                    "url": "https://github.com/vector-im/riot-web/archive/v1.5.1.tar.gz",
```
Comment (Erick555) [inline im.riot.Riot.json]: Files under `archive` subfolder (described as `Source code (tar.gz)` in github UI) are automatically generated by github and shouldn't be used as there is no guarantee they contain what upstream considered as release. Files under `releases/download/` (described as `riot-v1.5.1.tar.gz` in github UI) are proper releases.

@Mikaela @SISheogorath please keep in mind above for the future.

Comment (SISheogorath) [reply im.riot.Riot.json]: Looks like I missed that one! Only checked functionality.

I'll update this.

Comment (Erick555) [reply im.riot.Riot.json]: @SISheogorath in the meantime 1.5.3 was release (also deb) so you may update to it.

Comment (Mikaela) [reply im.riot.Riot.json]: Thanks, could it be added to CONTRIBUTING.md or somewhere?

Comment (Erick555) [reply im.riot.Riot.json]: The above recommendation is more about github in general rather than riot.

**Pull request merged**